src/tests/test_utils/common_test_utils/src/graph_comparator.cpp

```diff
@@ -294,7 +295,7 @@ class NodeAndOutputDescription {
                 return false;
             }
             for (size_t i = 0; i != result_shape.size(); ++i) {
-                const auto axis_multiplier = i == concat_desciption->m_axis ? num_iterations : 1;
+                const auto axis_multiplier = i == concat_desciption->m_axis ? static_cast<size_t>(num_iterations) : 1;
```

Comment:
```suggestion
                const auto axis_multiplier = static_cast<int64_t>(i) == concat_desciption->m_axis ? static_cast<size_t>(num_iterations) : 1;
```
But maybe using i64 is not required at all

Reply:
I'd rather change this function parameter type.